app/controllers/events_controller.rb

```diff
@@ -0,0 +1,35 @@
+class EventsController < ApplicationController
+  def new
```

Comment:
```ruby
 if current_user
      @event = current_user.events.build
    else
      redirect_to events_path
    end
```
This is okay to do some authentication, but rails have some filters to do this thing more proper way. You can play with that

![2019-06-13_161616](https://user-images.githubusercontent.com/6507903/59425452-c8fd6900-8df7-11e9-8a1c-9426ad307bf4.jpg)

Here you can get [more](https://guides.rubyonrails.org/action_controller_overview.html#filters)

Reply:
Fixed. Thanks for the review!